google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/DatasetInfo.java

```diff
@@ -142,6 +143,13 @@ public abstract Builder setExternalDatasetReference(
      */
     public abstract Builder setStorageBillingModel(String storageBillingModel);
 
+    /**
+     *
+     * Optional. Amount of time in hours that deleted or updated data will remain accessible
```

Comment:
Can you verify the description? I'm seeing slightly different description in the REST discovery document.

Reply:
Yes good idea, I've copied over the description from [API ](https://github.com/googleapis/google-api-java-client-services/blob/main/clients/google-api-services-bigquery/v2/2.0.0/com/google/api/services/bigquery/model/Dataset.java#L238)